react/features/invite/actions.js

```diff
@@ -98,7 +100,12 @@ export function invite(
         // For each number, dial out. On success, remove the number from
         // {@link invitesLeftToSend}.
         const phoneInvitePromises = phoneNumbers.map(item => {
-            const numberToInvite = item.number;
+            const {
+                dtmf,
+                phoneNumber: numberToInvite
+            } = splitPhoneNumberAndDTMF(item.number);
+
+            dtmf && dispatch(storePendingDTMF(dtmf));
```

Comment:
Maybe store only on successful dial, if the timing works out? If dial fails then does the dtmf remain in the store?

Reply:
Yeah, good point. I will check on that.